src/views/Articles/__tests__/PostForm.test.js

```diff
@@ -39,7 +43,27 @@ describe('pass', () => {
   it('test 1', () => {
     expect(wrapper).toMatchSnapshot();
   });
-
+  it('should call handleDrag', () => {
+    const wrapper = shallow(<PostForm {...props} />);
```

Comment:
Hello @Amoswachira ,
You could use the wrapper that had been declared in line 25 instead of declaring new ones in the unit tests

Reply:
@Ogutu-Brian let me work on that

Reply:
@Ogutu-Brian it is fixed